### PR TITLE
 Test Oracle with and without remarks reporting

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Comment by losipiuk** [comment]
2 commits just to make review easier. To be squashed.

**Comment by findepi** [comment]
> 2 commits just to make review easier. To be squashed.

This will obliterate the code history. Can we preserve it?

**Comment by losipiuk** [comment]
> > 2 commits just to make review easier. To be squashed.
> 
> This will obliterate the code history. Can we preserve it?

Sure. Yet the first commit does not work correctly (test class starts with `Base` which is not allowed)

**Comment by findepi** [comment]
> Sure. Yet the first commit does not work correctly (test class starts with `Base` which is not allowed)

I know. Maybe the first commit is a different rename.
Then the second commit is a rename once again.
Still two commits, but annotations are preserved.

The point is about:

> 2 commits just to make review easier. To be squashed.

This is really useful for review process but it is also useful later.

**Comment by losipiuk** [comment]
> > Sure. Yet the first commit does not work correctly (test class starts with `Base` which is not allowed)
> 
> I know. Maybe the first commit is a different rename.
> Then the second commit is a rename once again.
> Still two commits, but annotations are preserved.

Yeah I though about it but assumed that is too much fuss. 
Here you are :)

**Comment by findepi** [comment]
> Yeah I though about it but assumed that is too much fuss.

i bet this is subjective...

> Here you are :)

thanks!

File: presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestStandardOracleDistributedQueries.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.testing.sql.SqlExecutor;
+import io.prestosql.tpch.TpchTable;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.prestosql.testing.sql.TestTable.randomTableSuffix;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestStandardOracleDistributedQueries
```
Comment (findepi) [inline presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestStandardOracleDistributedQueries.java:26]: Standard -> '' ?

**Review by findepi — APPROVED** [review]
Thanks!

File: presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestStandardOracleDistributedQueries.java (hunk 1 above)
Comment (losipiuk) [reply presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestStandardOracleDistributedQueries.java]: Done

**Pull request merged**